.github/workflows/bitcoin-tests.yml

```diff
@@ -1,36 +1,55 @@
-name: stacks-bitcoin-integration-tests
+##
+## Bitcoin Integration Tests
+##
+
+name: Bitcoin Integration Tests
 
 # Only run when:
-#   - PRs are opened
+#   - PRs are (re)opened against master branch
+
 on:
   pull_request:
+    branches:
+      - master
+    types:
+      - opened
+      - reopened
```

Comment:
Does this mean that these tests wouldn't run for PRs against other branches? That would cut down on worker usage, but these tests are generally pretty helpful for PRs.

Reply:
Yeah, these tests should run on each PR before merging to `develop`, `next`, or `master`.

Reply:
Got it - yeah, that was a decision I made that i wasn't positive *should* be done. i'll revert back to how the tests are triggered today.

Reply:
https://github.com/wileyj/stacks-blockchain/commit/d8ce51e8d9a4990728bc43acf5e48c80a8079103